apps/judicial-system/web/src/routes/Prosecutor/Indictments/Indictment/IndictmentCount.tsx

```diff
@@ -61,6 +61,42 @@ function lawLabel(law: number[]): string {
   return `${law[0]}. mgr. ${law[1]}. gr. 77/2019`
 }
 
+function lawSort(law1: number[], law2: number[]) {
+  if (law1[0] < law2[0]) {
+    return -1
+  }
+  if (law1[0] > law2[0]) {
+    return 1
+  }
+  if (law1[1] < law2[1]) {
+    return -1
+  }
+  if (law1[1] > law2[1]) {
+    return 1
+  }
+  return 0
+}
+
+function legalArguments(lawsBroken?: number[][] | null) {
+  if (!lawsBroken || lawsBroken.length === 0) {
+    return ''
+  }
+
+  let legalArguments = `Telst háttsemi þessi varða við ${lawsBroken[0][1]}. mgr.`
```

Comment:
According to the design the "mgr." text should only come after the last law in a list that falls under the same gr., 

So the code would generate this:
Telst háttsemi þessi varða við 1. mgr., sbr. 2. mgr. 48. gr., sbr. 1. mgr. 95. gr. umferðarlaga nr. 77/2019.

But it should be like this:
Telst háttsemi ákærða varða við 1., sbr. 2. mgr. 50. gr., sbr. 1. mgr. 95. gr. umferðarlaga nr. 77/2019.

Reply:
Well spotted, will fix.